### ISSUE TITLE
Document state of iD's accessibility and setup compatability

### ISSUE THREAD

**Issue by quincylvania** [openstreetmap/iD#7713]
iD's goal has always been to make mapping as approachable as possible to as many people as possible. Much work has been done to make the interface friendly and intuitive to non-experts. But we have to keep in mind that there are many dimensions to accessibility—just because iD works great for some people doesn't mean it works well for everyone. Here are a few of the varied groups we should be constantly considering:

- People with visual impairments
- People with motor control impairments
- People with learning difficulties
- Speakers of languages other than English
- People unfamiliar with US/European culture
- Right-to-left layout users
- Mobile device users
- Touchscreen users
- People with limited internet connections
- …

iD does pretty well with some of these and poorly with others. I'd like to start a living document that details the accessibility of iD over a wide range of dimensions, with the intent of identifying and addressing problem areas. It's imperative we're proactive since mappers won't tell us what's wrong if they've already been alienated by the app.

**Comment by matkoniecz** [comment]
I would be happy to contribute - not very extensively but at least "speakers of languages other than English" is part where I may be useful.

I would consider adding "people completely unfamiliar with OSM" and "JOSM users" as both groups have some quirks.

Not sure is it within planned coverage but I would consider also "people with old hardware" - iD can be noticeably laggy even with relatively decent but old hardware, but at least it generally works.

Maybe also "people using browser other than Chrome"?

**Comment by quincylvania** [comment]
I've been working on this in a new [ACCESSIBILITY.md](https://github.com/openstreetmap/iD/blob/develop/ACCESSIBILITY.md) file. The name is short for convenience; the document involves a lot more.

**Comment by matkoniecz** [comment]
Would it be useful to mention #1646 in "Slow connection"?

**Comment by Adamant36** [comment]
Re "People with old hardware." I've been thinking for a while now that it would be cool of there was a way to toggle between basic and expert modes. Whatever those would entitled. Personally, I rarely use most of iD Editors more advanced features and it tends to slow my computer down a lot sometimes. Even though I have a fairly new hardware and a good internet connection. So a basic version that could run on older hardware  would be really great. 

BTW, I'm sure a lot of the preformance issues I have are because of Chrome's horrible memory management, but id still benefit from a basic version of iD Editor regardless. I can't imagine how it runs in Chrome on older machines.
